content/references/http-websocket-apis/public-api-methods/account-methods/account_info.md

```diff
@@ -204,7 +204,7 @@ The response follows the [standard format][], with the result containing the req
 |:-----------------------|:--------|:------------------------------------------|
 | `account_data`         | Object  | The [AccountRoot ledger object](accountroot.html) with this account's information, as stored in the ledger. |
 | `account_flags`        | Object  | The account's flag statuses (see below), based on the `Flags` field of the account. [New in: rippled 1.11.0][] |
-| `signer_lists`         | Array   | _(Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.)_ Array of [SignerList ledger objects](signerlist.html) associated with this account for [Multi-Signing](multi-signing.html). Since an account can own at most one SignerList, this array must have exactly one member if it is present. **Quirk:** In [API version 1](https://github.com/xrplf/rippled/blob/develop/API-CHANGELOG.md#v1-account_info-response), this field is nested under `account_data`. For this method, [Clio](https://github.com/XRPLF/clio) implements the API version 2 behavior where is field is **not** nested under `account_data`. |
+| `signer_lists`         | Array   | [API v1][]: _(Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.)_ Array of [SignerList ledger objects](signerlist.html) associated with this account for [Multi-Signing](multi-signing.html). Since an account can own at most one SignerList, this array must have exactly one member if it is present. The field is nested under `account_data`.<br>[API v2][]: Identical to API v1, but the field is returned in the root reponse instead. [Clio](https://github.com/XRPLF/clio) implements the API v2 behavior. |
```

Comment:
As above, avoid HTML in tables for Redocly compatibility. Also, I spotted a typo.

```suggestion
| `signer_lists`         | Array   | _(Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.)_ Array of [SignerList ledger objects](signerlist.html) associated with this account for [Multi-Signing](multi-signing.html). Since an account can own at most one SignerList, this array must have exactly one member if it is present. In [API v1][], the field is nested under `account_data`. In [API v2][], the field is returned in the root response instead. [Clio](https://github.com/XRPLF/clio) implements the API v2 behavior in all cases. |
```